### DISCUSSION TITLE
Collection of requirements, ideas, and features for Harbor 2.13

### DISCUSSION THREAD

**Discussion by Vad1mo** [goharbor/harbor#21054] [General]
We are collecting and ordering requirements for the 2.13 release. 

## Main Theme for 2.13
- Improving Audit logs
-  
-  

## The 2.13 Candidates so far
 - https://github.com/goharbor/harbor/labels/candidate%2F2.13.0

**Comment by kavyavvv** [comment]
can we get sentinel password support for external redis case ?

**Comment by kavyavvv** [comment]
can we get redis and sentinel tls support for external redis ?
currently we are using external redis and We have a requirement to enable tls for redis and sentinel and enable password for sentinel
But harbor does not support these

**Comment by coronabytes** [comment]
could we get 'mass' api endpoints for managing tags in multiple repos in a project as a single atomic operation?
like DELETE api/v2.0/projects/{project}/repositories/ ALL /artifacts/{tagToDelete}/tags/{tagToDelete}
and api/v2.0/projects/{project}/repositories/ ALL  /artifacts/{immutabletag}/tags and apply mutable tags on all of them?

reason: we have a CI/CD pipeline and would like to tag images on where they are actually rolled out.

**Reply by bupd** [reply]
you could try using Harbor-CLI for CI/CD pipelines. https://github.com/goharbor/harbor
